index.js

```diff
@@ -18,12 +18,16 @@ function notionPageContentIsLoaded() {
   return false
 }
 
-let pageLoadInterval = setInterval(() => {
-  if (notionPageContentIsLoaded()) {
-    setBlocksDirectionToAuto()
-
-    // TODO: If we can find a cleaner way to handle block additions/removals
-    // then clear out this interval
-    // clearInterval(pageLoadInterval)  
+chrome.storage.sync.get('mode', function (items) {
```

Comment:
I think I would prefer reading this config value separately and not tie the logic that's checking if Notion's page content is loaded to it.

Also one of the things I would love to do is make this work cross-browser and that's one of the reasons why I kind of slacked off on this kind of stuff but I don't want to tie the main logic to Chrome's API only if that makes sense. If we could instead have a module that reads configs (I believe there might also be a package/dependency that serves this purpose) and have that process be separate from our Notion-specific logic that would be great.

Reply:
I think using Mozilla's [WebExtension browser API Polyfill](https://github.com/mozilla/webextension-polyfill) is the best way to go at this point for cross-browser support, this is a starter template to use https://github.com/abhijithvijayan/web-extension-starter and this is an example of a cross-browser extension https://github.com/abhijithvijayan/kutt-extension, have a look at them and let me know if we should move forward in migrating to use the template or use the Polyfill directly without the rest of the starter setup.

p.s.: I don't really do JS much but i'm contributing in this project to refresh my js memory a little bit, so sorry if my PR had some weird js code. XD

Reply:
No worries man, I've gotten a little bit rusty myself but maintaining [are-you-es5](https://github.com/obahareth/are-you-es5) every once in a while really helps me out!

Reply:
```chrome.storage.sync``` should work fine with 

- Chrome
- Opera
- FF
- Possibility (new edge version didn't really test)

but for safari you will need to do some fallback work to use localStorage.

Note for FF: you will need to specify the id of extension in the manifest.js